src/cli/task.rs

```diff
@@ -182,6 +192,33 @@ pub struct Args {
     pub manifest_path: Option<PathBuf>,
 }
 
+fn print_heading(value: &str) {
+    let bold = console::Style::new().bold();
+    eprintln!("{}\n{:-<2$}", bold.apply_to(value), "", value.len(),);
+}
+
+fn print_tasks_per_env(envs: Vec<Environment>) -> io::Result<()> {
+    let mut writer = tabwriter::TabWriter::new(stdout());
+    for env in envs {
+        let formatted: String =
+            env.get_filtered_tasks()
+                .iter()
+                .sorted()
+                .fold(String::new(), |mut output, name| {
+                    let _ = write!(output, "{}, ", name.fancy_display());
```

Comment:
I don't think you should `write` here. You can `collect` and `join` on the sorted iterator.